tyk-data-plane/values.yaml

```
@@ -304,6 +304,53 @@ tyk-gateway:
     # if you want to send analytics to control plane instead of pump, change value to "rpc"
     analyticsConfigType: ""
 
+    # opentelemetry is used to configure opentelemetry for Tyk Gateway
+    opentelemetry:
+    # Used to enable/disable opentelemetry
+      enabled: false
+    # exporter is used to define the type of the exporter to sending data in OTLP protocol
+    # Valid values are "grpc" or "http"
+      exporter: grpc
+    # endpoint defines OpenTelemetry collector endpoint to connect to.
+      endpoint: localhost:4317
+    # A map of headers that will be sent with HTTP requests to the collector.
+    # It should be set to map of string to string
+      headers: {}
+    # Timeout for establishing a connection to the collector
+      connectionTimeout: 1
+    # Name of the resource that will be used to identify the resource.
+      resourceName: tyk
+    # Type of the span processor to use
+      spanProcessorType: batch
+    # Type of the context propagator to use
+      contextPropogation: tracecontext
+    # TLS configuration for the exporter.
+      tls: 
+      # Flag that can be used to enable TLS
+        enabled: false
+      # Flag that can be used to skip TLS verification if TLS is enabled
+        insecureSkipVerify: true
+      # Maximum TLS version that is supported.
+        maxVersion: 1.3
+      # Minimum TLS version that is supported
+        minVersion: 1.2
+      # Path to the cert file
+        certFileName:
+      # Path to the key file  
+        keyFileName:
```

Should we set it to an empty string, in order to indicate that the value is nil? What do you think?
```suggestion
        keyFileName: ""
```